### PR TITLE
Feature/new login

### PR DESCRIPTION
**Pull request by AdityaKhatri**
- Addresses https://github.com/the-deep/client/issues/1739,  https://github.com/the-deep/client/issues/1738, https://github.com/the-deep/client/issues/1742, https://github.com/the-deep/client/issues/1740, https://github.com/the-deep/client/issues/1741

- Depends on https://github.com/the-deep/server/pull/804
- Depends on https://github.com/the-deep/deep-ui/pull/65 (Plz deploy and use)

## Changes

* Add explore page
* Add login and register form
* Replace recaptcha with hcaptcha in old login and register form

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files, or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: src/views/PasswordReset/index.js [hunk 1]
```diff
@@ -75,13 +72,15 @@ function PasswordReset() {
     }, []);
 
     const handleFaramValidationFailure = useCallback((newFaramErrors) => {
+        captchaRef.current?.resetCaptcha();
```
Comment (tnagorra) [inline src/views/PasswordReset/index.js]: This is not required

File: src/constants/routes.js [hunk 2]
```diff
@@ -40,6 +40,16 @@ export const routes = {
...
+    exploreDeep: {
+        order: 10,
+        type: ROUTE.exclusivelyPublic,
+        redirectTo: '/',
+        path: '/new-ui/explore/',
+        hideNavbar: true,
+        loader: () => import('../newViews/Explore'),
+        links: noLinks,
+    }, // _ts('pageTitle', 'exploreDeep');
+
```
Comment (tnagorra) [inline src/constants/routes.js:52]: Please add this to viewsAcl as well. Even if it's empty object

File: src/newViews/Explore/ForgotPasswordModal/index.tsx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+    });
+
+    const handleSubmit = useCallback(
+        () => {
+            const { errored, error: err, value: val } = validate();
+            onErrorSet(err);
+            elementRef.current?.resetCaptcha();
+            if (!errored && isDefined(val)) {
+                triggerReset(val as ForgotPasswordFields);
+            }
```
Comment (tnagorra) [inline src/newViews/Explore/ForgotPasswordModal/index.tsx]: Shouldn't the resetCaptcha be called inside the if block

File: src/constants/viewsAcl.js [hunk 4]
```diff
@@ -16,7 +16,7 @@ const notAdmin = ({ isLoggedIn, isAdmin }) => (
     !isLoggedIn || !isAdmin
 );
 const notDevAndAdmin = ({ isLoggedIn, isDevMode, isAdmin }) => (
-    !isLoggedIn || (!isDevMode && !isAdmin)
+    isLoggedIn || (!isDevMode && !isAdmin)
```
Comment (tnagorra) [inline src/constants/viewsAcl.js]: Why did we change this?

File: src/newViews/Explore/LoginRegisterModal/LoginForm/index.tsx [hunk 5]
```diff
@@ -0,0 +1,318 @@
...
+                state: hidQuery.state,
+                tokenType: hidQuery.token_type,
+            };
+            hidLoginTrigger(params);
+        }
+    }, [hidLoginTrigger, location]);
+
+    useEffect(() => {
+        checkParamsFromHid();
+    }, [checkParamsFromHid]);
```
Comment (tnagorra) [inline src/newViews/Explore/LoginRegisterModal/LoginForm/index.tsx]: This looks a bit weird. Why define a separate named function checkParamsFromHid?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: src/components/ui/HCaptcha/index.tsx [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+            iconsContainerClassName={iconsContainerClassName}
+            inputSectionClassName={inputSectionClassName}
+            label={label}
+            labelContainerClassName={labelContainerClassName}
+            readOnly={readOnly}
+            uiMode={uiMode}
+            input={(
+                <HCaptcha
+                    ref={elementRef}
+                    // disabled={disabled || readOnly}
```
Comment (samshara) [inline src/components/ui/HCaptcha/index.tsx:80]: why is it commented out?

**Review by tnagorra — APPROVED** [review]

**Review by samshara — APPROVED** [review]

**Pull request merged**